mlir/test/Target/LLVMIR/rocdl.mlir

```diff
@@ -118,6 +118,25 @@ llvm.func @rocdl.ballot64(%pred : i1) -> i64 {
   llvm.return %0 : i64
 }
 
+llvm.func @rocdl.readlane(%src0 : f32, %src1: f64, %src2: i32, %src3: vector<2 x f32>) -> f32 {
```

Comment:
In addition to this, can we get a test in `mlir/test/Dialect/LLVMIR/rocdl.mlir` just to make sure the op parses/prints correctly?